harvard-de-montfort-university.csl

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - De Montfort University</title>
+    <id>http://www.zotero.org/styles/harvard-de-montfort-university</id>
+    <link href="http://www.zotero.org/styles/harvard-de-montfort-university" rel="self"/>
+    <link href="http://www.library.dmu.ac.uk/Images/Selfstudy/Harvard.pdf" rel="documentation"/>
+    <info>
+     <link href="http://www.zotero.org/styles/harvard1" rel="template"/>
+   </info>
```

Comment:
@ALRichards, our instructions include the `<info>` and `<info/>` tags just to show you where to add the template link, and you don't actually need to add those. The only thing you needed to add is the `<link .../>` element. Can you remove the info tags on the lines before and after?